evm/src/main/java/org/hyperledger/besu/evm/blockhash/BlockHashLookup.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.evm.blockhash;
+
+import org.hyperledger.besu.datatypes.Hash;
+import org.hyperledger.besu.evm.frame.MessageFrame;
+
+import java.util.function.BiFunction;
+
+/**
+ * Function that gets the block hash, passed in as part of TxValues.
```

Comment:
```suggestion
 * Function that gets the block hash.
```

Reply:
it's still passed as part of TxValues though: https://github.com/hyperledger/besu/blob/929945ad6c559d787a99c12e7f6c3dbcef6716f2/evm/src/main/java/org/hyperledger/besu/evm/frame/TxValues.java#L54